metrics/Cargo.toml

```diff
@@ -19,7 +19,8 @@ futures = { workspace = true }
 grandine_version = { workspace = true }
 helper_functions = { workspace = true, features = ['metrics'] }
 http_api_utils = { workspace = true }
-log = { workspace = true }
+logging = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order